From 329752895fb7fb946258508855d096e1cddbb3ed Mon Sep 17 00:00:00 2001
From: Ofer Heifetz <oferh@marvell.com>
Date: Wed, 21 Jun 2017 15:43:24 +0300
Subject: [PATCH 1722/2241] tools: nas: smb.conf changes to improve performance

- max xmit: increase improves the Samba read throughput
- server max protocol: change alignes the SMB version used with
  W8 and W10 OS

Change-Id: I89ed81f6c45eb770bc74d4908c903dd4395e5f8f
Signed-off-by: Ofer Heifetz <oferh@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/40669
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
---
 tools/nas/nas_init.sh | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/tools/nas/nas_init.sh b/tools/nas/nas_init.sh
index 4c5f7ed..29ab701 100755
--- a/tools/nas/nas_init.sh
+++ b/tools/nas/nas_init.sh
@@ -697,6 +697,7 @@ cat << EOF > ${SMB_CONF}
 	use sendfile = yes
 	dns proxy = no
 	max log size = 200
+	server max protocol = SMB3
 	log level = 0
 	socket options = IPTOS_LOWDELAY TCP_NODELAY
 	local master = no
@@ -708,7 +709,7 @@ cat << EOF > ${SMB_CONF}
 	printcap name = /dev/null
 	load printers = no
 	disable spoolss = yes
-	max xmit = 131072
+	max xmit = 1310720
 	disable netbios = yes
 	csc policy = disable
 	strict allocate = yes
-- 
2.7.4

